pkg/util/conntrack/conntrack.go

```diff
@@ -14,42 +14,53 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package util
+package conntrack
 
 import (
 	"fmt"
 	"net"
 	"strconv"
 	"strings"
 
+	"k8s.io/api/core/v1"
 	"k8s.io/utils/exec"
 )
 
 // Utilities for dealing with conntrack
 
+// NoConnectionToDelete is the error string returned by conntrack when no matching connections are found
 const NoConnectionToDelete = "0 flow entries have been deleted"
 
+// IsIPv6 returns true if the given ip address is a valid ipv6 address
 func IsIPv6(netIP net.IP) bool {
 	return netIP != nil && netIP.To4() == nil
 }
 
+// IsIPv6String returns true if the given string is a valid ipv6 address
 func IsIPv6String(ip string) bool {
 	netIP := net.ParseIP(ip)
 	return IsIPv6(netIP)
 }
 
+func protoStr(proto v1.Protocol) string {
```

Comment:
Why not just strings.ToLower() it and be done.  If the caller passes "ICMP", we should pass that thru to conntrack.

Reply:
Good point, done. i left the protoStr function as is, in case we want to add any checks in future.  Incorporated the other comments as well.